tests/RAGchain/utils/linker/test_dynamo_linker.py

```diff
@@ -1,16 +1,45 @@
 import pytest
 
 from RAGchain.utils.linker import DynamoLinker
+from RAGchain.utils.linker.base import NoIdWarning, NoDataWarning
 import test_base_linker
 
 
 @pytest.fixture
 def dynamo_db():
     dynamo_db = DynamoLinker(allow_multiple_instances=True)
-    dynamo_db.put_json(test_base_linker.TEST_IDS[0], test_base_linker.TEST_DB_ORIGIN)
     yield dynamo_db
     dynamo_db.flush_db()
 
 
-def test_get_json(dynamo_db):
-    assert dynamo_db.get_json(test_base_linker.TEST_IDS) == [test_base_linker.TEST_DB_ORIGIN]
+def test_get_json_UUID_UUID(dynamo_db):
+    dynamo_db.put_json(test_base_linker.TEST_UUID_IDS[0], test_base_linker.TEST_DB_ORIGIN)
+    assert dynamo_db.get_json(test_base_linker.TEST_UUID_IDS) == [test_base_linker.TEST_DB_ORIGIN]
+
+
+def test_get_json_UUID_STR(dynamo_db):
+    dynamo_db.put_json(test_base_linker.TEST_UUID_IDS[0], test_base_linker.TEST_DB_ORIGIN)
+    assert dynamo_db.get_json(test_base_linker.TEST_STR_IDS) == [test_base_linker.TEST_DB_ORIGIN]
+
+
+def test_get_json_STR_UUID(dynamo_db):
+    dynamo_db.put_json(test_base_linker.TEST_STR_IDS[0], test_base_linker.TEST_DB_ORIGIN)
+    assert dynamo_db.get_json(test_base_linker.TEST_UUID_IDS) == [test_base_linker.TEST_DB_ORIGIN]
+
+
+def test_get_json_STR_STR(dynamo_db):
+    dynamo_db.put_json(test_base_linker.TEST_STR_IDS[0], test_base_linker.TEST_DB_ORIGIN)
+    assert dynamo_db.get_json(test_base_linker.TEST_STR_IDS) == [test_base_linker.TEST_DB_ORIGIN]
```

Comment:
duplicate tests
Make test code at base and just replace linker instance for testing.